client/lib/create-selector/test/index.js

```diff
@@ -1,22 +1,19 @@
 /**
  * External dependencies
  */
-import Chai, { expect } from 'chai';
+import { expect } from 'chai';
 import filter from 'lodash/filter';
 import sinon from 'sinon';
-import sinonChai from 'sinon-chai';
 
 /**
  * Internal dependencies
  */
 import createSelector from '../';
 
-describe( '#createSelector', () => {
+describe( 'index', () => {
 	let selector, getSitePosts;
 
 	before( () => {
-		Chai.use( sinonChai );
```

Comment:
Do we need to use sinonChai globally? Does it impact assertions in general? Does this test suite use it at all?